packages/cf-component-form/src/FormLabel.js

```diff
@@ -1,24 +1,23 @@
 import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
 
-class FormLabel extends React.Component {
-  render() {
-    let className = 'cf-form__label';
+const styles = ({ theme, hidden }) => ({
+  fontSize: theme.fontSize,
+  display: hidden ? 'none' : 'block',
+  marginBottom: theme.marginBottom,
+  color: theme.color
+});
 
-    if (this.props.hidden) {
-      className += ' cf-form__label--hidden';
-    }
-
-    return (
-      <label className={className}>
-        {this.props.children}
-      </label>
-    );
-  }
-}
+const FormLabel = ({ children, className }) => (
```

Comment:
We'd like to avoid using refs, but can't. Refs also isn't just something used by DOM manip libs too, tho that's a big issue because tooltip and such. Let's wait til fiber.